component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java

```diff
@@ -759,6 +759,9 @@ public void autoDiscoverPlugins(final boolean callers, final boolean classpath)
                         Thread.currentThread().getContextClassLoader().getResources("TALEND-INF/dependencies.txt");
                 while (componentMarkers.hasMoreElements()) {
                     File file = Files.toFile(componentMarkers.nextElement());
+                    if (file == null) {
+                    	continue;
```

Comment:
Also, the modification is harmless but did you made some tests?

Reply:
It's not hamless, it will result in https://jira.talendforge.org/browse/TESB-27788 . However, we have a fix for the latter.

Reply:
I also run a checkstyle and formatting but github sometimes displays incorrectly but I don't know how to fix this from github side.

Reply:
> It's not hamless, it will result in https://jira.talendforge.org/browse/TESB-27788 . However, we have a fix for the latter.

That's not the same purpose, you're talking about your blocking issue. Me, talking about the code change repercutions and side effects.

Reply:
> I also run a checkstyle and formatting but github sometimes displays incorrectly but I don't know how to fix this from github side.

Normally, you should always run the package or install lifecycle before committing in this repository. This also applies code formatting. Your indentation level is incorrect so you did not run it apparently.

Reply:
I'll rerun